sycl/test/inline-asm/asm_no_operands.cpp

```diff
@@ -25,10 +25,11 @@ int main() {
   Queue.submit([&](cl::sycl::handler &cgh) {
     // Executing kernel
     cgh.parallel_for<no_operands_kernel>(
-        NumOfWorkItems, [=](cl::sycl::id<1> WIid) [[cl::intel_reqd_sub_group_size(8)]] {
+        NumOfWorkItems, [=](cl::sycl::id<1> WIid)
+                            [[intel::reqd_sub_group_size(8)]] {
 #if defined(INLINE_ASM) && defined(__SYCL_DEVICE_ONLY__)
-          asm("barrier");
+                              asm("barrier");
 #endif
-        });
+                            });
```

Comment:
These changes seem to be unrelated.

Reply:
Yes, this is unrelated. Clang-format complained about this.  I will change this to the original.